### DISCUSSION TITLE
Any ETA for STABLE version with PHP 8 support?

### DISCUSSION THREAD

**Discussion by sergyu91ma** [deployphp/deployer#2391] [Help needed]
Hi,

Do we have any ETA for STABLE version with PHP 8 support?

Thanks.

**Comment by antonmedv** [comment]
V7 already supports php8.

**Comment by antonmedv** [comment]
It mostly stable.

**Reply by sergyu91ma** [reply]
Yes, but it`s not released.
I use https://github.com/lorisleiva/laravel-deployer and they don't want to include beta versions. 
Check this: 
https://github.com/lorisleiva/laravel-deployer/issues/222

**Reply by antonmedv** [reply]
Deployer v7 is acually almost same as laravel-deployer. Check it out.
